### DISCUSSION TITLE
Same result for -t and -n flag

### DISCUSSION THREAD

**Discussion by thalesmaoa** [elementary/terminal#744] [Q&A]
I think this is more like a bug report, but I can't make terminal open a new windows and execute a program. I really like julia, but I can't create a shortcut to open it.

As an example, this never works:

```
io.elementary.terminal -e ls -n
```
It always opens two tabs and don't save the julia icon such as uxterm.

![image](https://github.com/elementary/terminal/assets/14074573/36aa042e-f889-48f9-8238-c232ff928b12)

**Comment by jeremypw** [comment]
Have you tried `io.elementary -ne ls`?  The issue of the unwanted default tab in addition to the one performing the command is recognised elsewhere and a fix has been proposed #743.

**Reply by thalesmaoa** [reply]
Yes! I did. New windows with two tabs. After checking your reference, I've noticed that the commit isn't in the master branch.
